spec/indexers/descriptive_metadata/event_date_creation_spec.rb

```diff
@@ -73,363 +72,349 @@
     JSON
   end
 
-  describe 'date mappings from Cocina to Solr' do
-    describe 'origin_info_date_created_tesim' do
-      # Creation date
-      let(:doc) { indexer.to_solr }
-
-      context 'when date.type creation and date.status primary' do
-        # Select date.type creation with date.status primary
-        let(:description) do
-          <<~JSON
-            "title": [
-              {
-                "value": "Title"
-              }
-            ],
-            "event": [
-              {
-                "date": [
-                  {
-                    "value": "1900",
-                    "type": "creation",
-                    "status": "primary"
-                  }
-                ]
-              }
-            ]
-          JSON
-        end
+  describe 'date mappings from Cocina to Solr originInfo_date_created_tesim' do
+    context 'when date.type creation and date.status primary' do
+      let(:description) do
+        {
+          title: [
+            {
+              value: 'Title'
+            }
+          ],
+          event: [
+            {
+              date: [
+                {
+                  value: '1900',
+                  type: 'creation',
+                  status: 'primary'
+                }
+              ]
+            }
+          ]
+        }
+      end
 
-        xit 'populates origin_info_date_created_tesim' do
-          expect(doc).to include('origin_info_date_created_tesim' => '1900')
-        end
+      it 'uses date' do
```

Comment:
I switched to using a more descriptive statement in the spec description, though this is the worst example.